utils/os_env.go

```diff
@@ -0,0 +1,16 @@
+package utils
+
+import (
+	"fmt"
+	"os"
+)
+
+// GetEnv returns the value of the environment variable named by the key
+// and sets the environment variable up to be used in the remote runner
+func GetEnv(key string) string {
+	val := os.Getenv(key)
+	if val != "" {
```

Comment:
What if `val` is already prefixed with `TEST_`?

Reply:
Shouldn't matter. For example if you wanted to propagate TEST_SOMETHING to propagate it you want it to be TEST_TEST_SOMETHING. Or am I missing something else?

Reply:
I can add logic to make sure the prefixed isn't already set and send an error if so. Might prevent someone from bonking against this in a weird way in the future.

Reply:
Alright we throw an error now if there is an existing prefixed key.